torch/_torch_docs.py

```diff
@@ -2931,6 +2931,25 @@ def merge_dicts(*dicts):
     tensor([False, True, False])
 """)
 
+add_docstr(torch.isreal,
+           r"""
+isreal(input) -> Tensor
+
+Returns a new tensor with boolean elements representing if each element of :attr:input is real-valued or not.
```

Comment:
This is good but you need the backticks around "input" for this (and the reference below) to format properly:

```
:attr:`input`
```